main/bind/APKBUILD

```diff
@@ -68,6 +74,13 @@ prepare() {
 	default_prepare
 	cd "$builddir"
 
+	# Fetch and verify named.root
+	gpg --import < "$srcdir/verisign-grs-nstld-key.asc"
+	rm -rf named.root named.root.sig
+	curl -RLO 'https://www.internic.net/domain/named.root{.sig,}'
```

Comment:
Slight pedant note, but the file references itself as named.cache. All three (named.cache, named.root, db.cache) all have the same signature etc and are identical, should we use that internal naming? There's no *material* difference, just curious mainly.

Reply:
fair enough, I myself also have no idea behind the naming - I've always called it root.hints or something meaningful like that myself.